source/code/source/playfab/PlayFabAndroidHttpPlugin.cpp

```diff
@@ -431,7 +431,7 @@ namespace PlayFab
 
         // Call SetUrl
         {
-            auto requestUrl = GetUrl(requestTask);
+            const std::string& requestUrl = GetUrl(requestTask);
```

Comment:
>const std::string& [](start = 12, length = 18)

FWIW calling GetUrl constructs a string so using `auto` here (and similarly for GetRequestHeaders below) doesn't actually result in a string copy and using const std::string& doesn't save us any string constructions. I don't have a preference one way or the other what you do here but it may be misleading to some that the variable is a reference type when GetUrl isn't returning a reference.

Reply:
also applies to your GetCallback() below since that also returns a Callback value and not a reference 

---
In reply to: [340919974](https://github.com/PlayFab/XPlatCoreTemplate/pull/33#discussion_r340919974) [](ancestors = 340919974)

Reply:
noticing that GetCallback didn't change in the latest iteration. intentional?

---
In reply to: [340920452](https://github.com/PlayFab/XPlatCoreTemplate/pull/33#discussion_r340920452) [](ancestors = 340920452,340919974)